languages/ruby/exercises/concept/basics/.meta/design.md

```diff
@@ -0,0 +1,49 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of programming in Ruby.
+
+## Learning objectives
+
+- Know what a variable is.
+- Know how to define a variable.
+- Know how to update a variable.
+- Know how to define a method.
+- Know how to return a value from a method.
+- Know how to call a method.
+- Know how to call a method with named parameters.
+- Know that methods must be defined in classes.
+- Know how to define an integer.
+- Know how to use mathematical operators on integers.
+- Know how to use constants
+
+## Out of scope
+
+- Memory and performance characteristics.
+- Method overloads.
+- Lambdas.
+- Optional parameters.
+- Organizing methods in namespaces.
+- Visibility.
+
+## Concepts
+
+The Concepts this exercise unlocks are:
+
+- `basics`: know what a variable is; know how to define a variable; know how to update a variable; know how to use type inference for variables; know how to define a method; know how to return a value from a method; know how to call a method; know that methods must be defined in classes; know about the `public` access modifier; know about the `static` modifier; know how to define an integer; know how to use mathematical operators on integers; know how to define an integer; know how to use mathematical operators on integers.
```

Comment:
Can we bullet each of the points here?  Would that make it easier to iterate?

```suggestion
- *basics*: know
  - what a variable is
  - how to define a variable
  - how to update a variable
  - how to use type inference for variables
  - how to define a method
  - how to return a value from a method
  - how to call a method
  - that methods must be defined in classes
  - about the `public` access modifier
  - about the `static` modifier
  - how to define an integer
  - how to use mathematical operators on integers
  - how to define an integer
  - how to use mathematical operators on integers.
```

Reply:
We can. They're mostly written the way they are as that makes them easier to add to a table without causing too much lines to be taken up: https://github.com/exercism/v3/blob/master/languages/csharp/reference/README.md#concept-interpretation

Reply:
Alright.  Either way is good with me.

Reply:
I'll leave this for now as I don't want it to block this PR, but we can debate it in a seperate PR if required.